### DISCUSSION TITLE
addMotifAnnotations

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#995] [Questions / Documentation]
Can we add multiple motif databases together?
I wanted to increase the number of annotations detected.

Thanks!

**Comment by rcorces** [comment]
You can create your own motif set as you wish. See the `motifPWMs` argument to `addMotifAnnotations()`
<html>
<body>


motifPWMs | A custom set of motif PWMs as a PWMList for adding motif annotations.
-- | --



</body>
</html>

**Reply by sylestiel** [reply]
Thanks!!

**Reply by sylestiel** [reply]
Could you provide some details on how to generate a PWM List for custom motif list that utilizes multiple chromVAR motif databases (preferably without redundancy)?

**Comment by sylestiel** [comment]
Could you provide some details on how to generate a PWM List for custom motif list that utilizes multiple chromVAR motif databases (preferably without redundancy)?

**Reply by rcorces** [reply]
Sorry - I dont have code for this easily handy. But it shouldnt be too hard. Maybe just look into the TFBStools documentation. Its mostly basic list manipulation in R.

**Reply by sylestiel** [reply]
Okay. Thanks!

**Reply by xmliu01** [reply]
Did you add a custom motif list successfully? How to provide the motif list in PWM List format?
